src/randomMeasures/RandomMeasures.jl

```diff
@@ -0,0 +1,14 @@
+module RandomMeasures
+
+using ..Core, ..Core.VarReplay, ..Utilities
+using Distributions
+using LinearAlgebra
+using StatsFuns: logsumexp
+
+import Distributions: sample, logpdf
+import Base: maximum, minimum, rand
+
+# include concrete implementations
+include("rpm.jl")
```

Comment:
it's ok to directly include the contents of `rpm.jl` here since it's not a huge amount of code. It also makes future maintenance easier since it will only involve one file.

Reply:
Hm, maybe we can move the generic bits here and move the implementation related to random partitions into a file `randomPartitions.jl` or so. This way it would be easier to extend the code later on.

Reply:
I don't have a strong preference here - but have seen many Julia basic libraries contained in a single file separated by sections. I think that makes sense cause have several small files doesn't add any flexibility, right?

Reply:
Fair enough.